### ISSUE TITLE
Insufficient hover effect on console Clear button

### ISSUE THREAD

**Issue by avengsum** [processing/p5.js-web-editor#2592]
### p5.js version

v1.8.0

### What is your operating system?

Windows

### Web browser and version

119.0.6045.124

### Actual Behavior

while hovering the cursor over the Clear button the background color is not visible properly 
![Screenshot 2023-11-11 143108](https://github.com/processing/p5.js-web-editor/assets/123776031/d3ab489c-eed9-4cef-befe-a97ad26aa8e4)

### Expected Behavior

we should increase its intensity so that we can see properly

### Steps:
1. Go to p5.js editor
2. Try to hover over the clear button

**Comment by Garima3110** [comment]
Thanks @avengsum . I would like to work on this issue..

**Comment by mhsh312** [comment]
are you saying the hover shadow should be bigger?

**Comment by Garima3110** [comment]
> are you saying the hover shadow should be bigger?

well, i think @avengsum is  referring to using a darker color for the hover effect so that it is visible clearly.

**Comment by lindapaiste** [comment]
Let's discuss what we think it should look like when hovered!

The current effect is that the text color gets a little darker.  There is no change to the background color.  That is similar to the chevron open/close icon button next to it, but the color change is much more noticeable on the icon because the lines are thicker.

We could make the text bold when hovered.  That might be weird because there's a little bit of movement due to bold text being larger/wider.  But it definitely makes the hover very noticeable.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/6b65ccdb-47d3-466f-9874-fab5cdf5d39a)

We could change the text color to the primary color (pink) like we do on menu items. This is not particularly readable but it shows that you are interacting with something clickable.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/9d8ccba5-4c20-40b0-ad20-aec94ff1b392)

A bolder move would be to make the whole background pink on hover.  Similar to the button for the sidebar files.  This involves some tweaks to the border-radius, padding, etc.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/e67bf432-153c-4719-928e-d2f1b9c9deca)
If we do this, then I think we would want a stronger effect on the chevron open/close button that is next to it as well.  So this could involve a lot of changes.

The CSS I'm playing with in dev tools for that mockup is 
```
.light .preview-console__clear:hover {
    background: #ed225d;
    color: white;
    padding: 2px 6px;
    border-radius: 2px;
    margin-right: 4px;
}
```

**Comment by mhsh312** [comment]
I think what we should do is:
-Make the text `#ed225d` on hover
-Also add a #ed225d solid shadow to it so that it increases visibility (kinda like bold) but it doesn't look jarring like when making it bold as shadow doesn't move the text like bold does.


That way we are following the theme of p5 and making sure readable. 

Code:
```
.light .preview-console__clear:hover {
    color: #ed225d;
    text-shadow: 0.25px -0.25px 0px #ed225d, 0.25px 0.25px 0px #ed225d, -0.25px 0.25px 0px #ed225d, 0.25px -0.25px 0px #ed225d;
}
```
Output:
![image](https://github.com/processing/p5.js-web-editor/assets/135870090/4f939915-5aac-437b-8ddd-ecd244832372)

If not, I think changing the color to pink and making the text bold is the best option.

**Comment by yashpandey06** [comment]
I will suggest changing of the text color rather than  making it whole background colored  .

**Comment by PiyushChandra17** [comment]
Well, I think the second option what @lindapaiste proposed ie. just changing the text color will work fine.

**Comment by ujjwaldubey1** [comment]
I agree with you @PiyushChandra17  because @lindapaiste  had previously said that if we make the text bolder then the size feels it is increasing so the text bold thing don't work for us for this time so the background color change would be the great option to do.

**Comment by Harshit-7373** [comment]
Even I feel the same @PiyushChandra17 and @ujjwaldubey1 , Changing only the text color will be the best option.
